README.md

```diff
@@ -106,16 +111,22 @@ fh.run()
 
 ## Supported Channels
 
-Cryptofeed supports the following channels:
+Cryptofeed supports the following channels from exchanges:
 
 * L2_BOOK - Price aggregated sizes. Some exchanges provide the entire depth, some provide a subset.
 * L3_BOOK - Price aggregated orders. Like the L2 book, some exchanges may only provide partial depth.
-* TRADES - Note this reports the taker's side, even for exchanges that report the maker side
+* TRADES - Note this reports the taker's side, even for exchanges that report the maker side.
 * TICKER
 * VOLUME
 * FUNDING
 * BOOK_DELTA - Subscribed to with L2 or L3 books, receive book deltas rather than the entire book on updates. Full updates will be periodically sent on the L2 or L3 channel. If BOOK_DELTA is enabled, only L2 or L3 book can be enabled, not both. To receive both create two `feedhandler` objects. Not all exchanges are supported, as some exchanges send complete books on every update.
-* OPEN_INTEREST - Open interest data
+* OPEN_INTEREST - Open interest data.
+
+Aggregated data from provider is available in channel:
+
+* TRANSACTIONS - On-chain transactions.
+* PROFILE - current aggregated price, market cap, volume (in USD, BTC or ETH currency), total and circulating supply,
```

Comment:
I'm trying to think of a better name than profile. Profile seems like the name that would be used for private account information. The information here is more summary or generic statistics. What about INFORMATIONAL? or something like that, any thoughts?

Reply:
Hi @bmoscon 
I am fine with INFORMATIONAL if this is ok, english-ly speaking :)  I am french, so don't ask me which word could be more suited for this :)
Instead of PROFILE, it could also be COIN_PROFILE
Coingecko calls this MARKET, but I am not fond of that.
Within this data, there are sentiment data as well.
INFORMATIONAL makes sense yes, why not.

Reply:
I'm fine with MARKET, MARKET_INFO, or COIN_PROFILE - take your pick :)

Reply:
@bmoscon, so let this be MARKET_INFO ! :)

Reply:
ok, done!